Source/WebKit/NetworkProcess/Classifier/ResourceLoadStatisticsDatabaseStore.cpp

```diff
@@ -543,6 +543,7 @@ void ResourceLoadStatisticsDatabaseStore::openAndUpdateSchemaIfNecessary()
 
     if (!vectorEqualsArray<String, ASCIILiteral, 13>(columns, currentSchemaColumns)) {
         close();
+        ITP_RELEASE_LOG_ERROR(m_sessionID, "%p - ResourceLoadStatisticsDatabaseStore::openAndUpdateSchemaIfNecessary failed schema check; will create new database", this);
```

Comment:
Our intention is to change `ResourceLoadStatistics` to `TrackingPrevention`. Would it make sense to reflect that in this log statement already to future proof it?

Reply:
This is the current format for all ITP_RELEASE_LOG_ERROR. I think we can stick to this for now for consistency and change all of them when we change ResourceLoadStatistics to TrackingPrevention.